docs/projects/ad353xr/index.rst

```diff
@@ -0,0 +1,191 @@
+.. _ad353xr:
+
+AD353XR HDL project
+================================================================================
+
+Overview
+--------------------------------------------------------------------------------
+
+The :adi:`AD3530R`/ :adi:`AD3530` are low power, 8-channel, 16-bit, buffered
+voltage output, digital-to-analog converters (DACs) that include a gain bit field,
+resulting in a full-scale output of 2.5 V (gain = 1) or 5 V (gain = 2) for a
+reference voltage of 2.5 V. The :adi:`AD3530R` has an on-chip, buffered, 2.5 V
+reference available at the VREF pin, capable of sourcing external loads up to
++5 mA.
+
+Each DAC channel has its own Input register and DAC register. The DAC Register 
+stores digital code equivalent to the DAC output voltage while the Input Register
+acts as a temporary staging register before being passed on the DAC Register. With
+the LDAC function, one or more DAC registers could be updated in parallel with
+the data held in the Input Register. The DAC registers can also be directly written
+to, in which the corresponding output updates immediately without an LDAC. 
+
+The :adi:`AD3530R`/ :adi:`AD3530` contains eight buffered voltage output DAC 
+channels capable of sourcing and sinking up to 40 mA of current.
+
+The :adi:`AD3530R`/ :adi:`AD3530` contains a 27:1 multiplexer which could
+output a voltage on the MUX_OUT pin that is a representative of
+either the output voltage or output current of a chosen channel, or
+the internal die temperature of the device.
+
+
+Applications:
+
+* Optical transceivers
+* Test and measurement
+* Industrial automation
+* Data acquisition systems
+
+Supported boards
+-------------------------------------------------------------------------------
+
+- :adi:`EVAL-AD3530RARDZ`
+
+Supported devices
+-------------------------------------------------------------------------------
+
+- :adi:`AD3530`
+- :adi:`AD3530R`
+
+Supported carriers
+-------------------------------------------------------------------------------
+
+- :xilinx:`ZedBoard <products/boards-and-kits/1-8dyf-11.html>` on FMCs
+- :xilinx:`Cora Z7-07S <products/boards-and-kits/1-1qlaz7n.html>` on GPIOs
+- :intel:`DE10-Nano <content/www/us/en/partner/showcase/offering/a5b3b0000004cbwAAA/de10nano-kit.html>` on GPIOs
```

Comment:
+ specify the PMOD for Cora and DE10-Nano

```suggestion
- :xilinx:`ZedBoard <products/boards-and-kits/1-8dyf-11.html>` on FMC
- :xilinx:`Cora Z7-07S <products/boards-and-kits/1-1qlaz7n.html>` on GPIOs
- :intel:`DE10-Nano <content/www/us/en/partner/showcase/offering/a5b3b0000004cbwAAA/de10nano-kit.html>` on GPIOs
```

Reply:
Ohh, sorry, I thought it was on PMOD... Then you can leave things as they are now.